.changeset/big-spiders-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
```

Comment:
![image](https://github.com/user-attachments/assets/a4f71eca-c604-4437-b5a1-f3babd770b76)
I think we should start to use either a console.log or use a text element that says Disclosure is {isExpanded}

Reply:
probably not a console log. but I'm confused as to why, like what's your thought process behind this?

Reply:
To show that it's controlled, otherwise you looking at the sample only you need to trust that it works.